pkg/windows/calico.go

```diff
@@ -173,7 +172,7 @@ func getDefaultConfig(config *CNIConfig, dataDir string, nodeConfig *config.Node
 		Platform:              platformType,
 		StartUpValidIPTimeout: 90,
 		LogDir:                "",
-		IP:                    "autodetect",
+		IP:                    nodeConfig.AgentConfig.NodeIP,
```

Comment:
we're unconditionally hardcoding node IP here? If this is unset by the user, should it default back to autodetect?